### DISCUSSION TITLE
Offline map

### DISCUSSION THREAD

**Discussion by meroua** [Mapsui/Mapsui#1271] [General]
I am trying to load offline maps using mapsui latest version. My project is .Net.Core 3.1
You can find the project at
https://github.com/meroua/WpfApp1

When I used the world.mbtiles I can see the map but I can't zoom in.
When I used another map, i.e. cymap.mbtitles I got many exceptions.

What I am doing wrong?

**Comment by pauldendulk** [comment]
If you run the wpf sample and select the data category there are two mbtiles samples. 
- 1 MbTiles. Shows a full map. It is not possible to zoom in deep because there are just a few layers in that mbtiles file. There is no more detailed zoom level.
- 2 MbTiles overlay. Has a small tile set. It is located in Spain, you have to know where it is. You have to zoom into: Torrejón de Ardoz. We have to fix the sample so it shows that place on startup.

Start from one of these working sample and go from there.

**Reply by meroua** [reply]
With the world.mbtiles it is working, but when I used another mbtile file I got exception 
"SkiaSharp.pdb contains the debug information required to find the source for the module SkiaSharp.dll"
"System.ArgumentNullException: 'Value cannot be null. '"
"SkiaSharp.pdb not loaded"

But when I used the same mbtile file in mapsui version "1.3.2.0" it is working fine.

**Reply by pauldendulk** [reply]
Looks like there is something going wrong with our debug information as well.

Could you try to load the file directly with BruTile. You could checkt out the [BruTile](https://github.com/BruTile/BruTile) repository and replace the mbtiles in one of the samples with yours.
